### ISSUE TITLE
First Person Body does not show correctly while flying and swimming

### ISSUE THREAD

**Issue by bfreezie** [tr7zw/FirstPersonModel#516]
MC 1.20.1
Fabric 16.5

Elytra gliding completely removes the first person body, and swimming seems to hide the body but hardcode the first person arm swimming animation. I understand that this is ideal for most use cases, however, while trying to use FPB in conjunction with Customizable Player Models it is frustrating that FPB will overwrite my ability to make custom animations that are visible in FP.

**Comment by tr7zw** [comment]
Correct, that's by design. Customizable Player Models is not officially supported (if it works, good, if not, then it doesn't), and I can't think of anything where you actually want the torso to clip the camera while swimming/gliding? For gliding I may get having a custom pose where seeing the arm might be a thing, but I think right now the entire playermodel and camera are just not lined up flying, so it would glitch all over the place if you could force it visible.

(Also the mod is called FirstPersonModel, not First Person Body^^)

**Comment by bfreezie** [comment]
Gotcha, that's kind of what I figured, and thanks for responding! Since I can control the torso and arm position via CPM clipping wouldn't be a problem if could just make it visible. Perhaps CPM can make an override toggle of some kind for this, since CPM already supports a custom animation for when your mod is present.

Anyway, thanks for a great mod, I love it and it's hard to play without it.

(Also, apologies for my brain's inability to remember the mod's name. Every single time I go to google it I get it wrong, lol)
